src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java

```diff
@@ -100,8 +100,14 @@ public static void initialize(ModelDao modelDao) {
     }
 
     private static float[] ObjectsToFloats(List<Object> objs) {
+        if (Objects.isNull(objs)) {
```

Comment:
Is it okay to empt list?

Reply:
Good point. we should look into this case.

Reply:
Added check for empty list